pkg/occlient/projects.go

```diff
@@ -0,0 +1,236 @@
+package occlient
+
+import (
+	"fmt"
+	"time"
+
+	projectv1 "github.com/openshift/api/project/v1"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog"
+)
+
+// GetProject returns project based on the name of the project.Errors related to
+// project not being found or forbidden are translated to nil project for compatibility
```

Comment:
Is it just me or is it really unclear/confusing as to what this function does based on the comment?

Reply:
fixed